lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1524,10 +1523,12 @@ defmodule Phoenix.LiveViewTest do
     quote bind_quoted: binding() do
       case reason do
         {:error, {:live_redirect, opts}} ->
+          opts = Phoenix.LiveViewTest.__ensure_signed_flash__(@endpoint, opts)
           {conn, to} = Phoenix.LiveViewTest.__follow_redirect__(conn, to, opts)
           live(conn, to)
 
         {:error, {:redirect, opts}} ->
+          opts = Phoenix.LiveViewTest.__ensure_signed_flash__(@endpoint, opts)
```

Comment:
Instead of adding a new callback, can you please move this logic to `__folow_redirect__`? You can pass the `@endpoint` to follow redirect too. Other than that, just a test and we are good to go!